nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java

```diff
@@ -1834,7 +1834,7 @@ public void setSampleSchedule(SampleSchedule sampleSchedule) {
     }
 
     private List<String> scanPulseSequence(Path path) throws IOException {
-        return Files.readAllLines(path);
+        return Files.readAllLines(path, Charset.forName("ISO-8859-1"));
```

Comment:
I think sonar lint wants this change made
```suggestion
        return Files.readAllLines(path, StandardCharsets.ISO_8859_1);
```